.github/.config/sugar-cli-test.amman.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const base = require('../../.base-ammanrc.js');
+
+const validator = {
+  ...base.validator,
+  programs: [],
+  accountsCluster: 'https://devnet.genesysgo.net',
+  accounts: [
```

Comment:
To not break CI due to changes on devnet you may wanna include the program `.so` files instead.
That is unless you _do_ want to break if devnet becomes incompat with what sugar assumes.
But then maybe pulling from main-net is the better thing to do?

Not sure .. either way pulling it from either net prevents us from testing features that haven't been deployed yet.

Reply:
Yeah, I think this is a more philosophical question w.r.t. what we want to test against. I can see an argument for all three. If testing against local changes, we will need to produce a `.so` file for candy machine and token metadata unless those `.so` files were previously cached somewhere, like AWS.

Maybe @austbot can weigh in here.

Reply:
Another path is to provide the option to choose build and use local, use devnet, or use mainnet?